config/webpack.config.js

```diff
@@ -48,6 +48,13 @@ var config = {
       {
         test: /(\.png|\.gif)$/,
         loader: "url-loader?limit=32767"
+      },
+      {
+        test: /\.scss$/,
+        loader: ExtractTextPlugin.extract(
+          'style', // The backup style loader
```

Comment:
shouldn't this be 'style-loader'? Or is the loader above (for regular css) incorrect?